### PR TITLE
Add PHP 7.1 and 7.2 to Travis test matrix

### PR DESCRIPTION
**Pull request by emirb**

### CODE REVIEW

**Comment by emirb** [comment]
All tests are passing now. I've changed configuration to use Ubuntu 12.04 LTS (precise) because it's the one that still supports 5.3. Also, changed to `sudo: false` to make use of containerised infrastructure in Travis.

**Comment by emirb** [comment]
@serbanghita bump

**Comment by JayBizzle** [comment]
You could setup the testing matrix like this to only uses the Precise distro on the 5.3 tests. Also, nice to add the nightly builds with allowed failures and a fast finish

```yaml
matrix:
  include:
    - php: 5.3
      dist: precise
    - php: 5.4
    - php: 5.5
    - php: 5.6
    - php: 7.0
    - php: 7.1
    - php: 7.2
    - php: nightly
  allow_failures:
    - php: nightly
  fast_finish: true
```

**Comment by emirb** [comment]
Thanks @JayBizzle, I updated `.travis.yml` accordingly.

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,20 @@
 sudo: false
 dist: precise # because it's still supporting PHP 5.3
```
Comment (JayBizzle) [inline .travis.yml]: You can take this line out now you have specified it in the matrix

**Comment by emirb** [comment]
Can this be merged @serbanghita?

**Comment by serbanghita** [comment]
@emirb thank you 🍻

**Pull request merged**